corehq/apps/reports_core/static/reports_core/js/bootstrap3/maps.js

```diff
@@ -2,11 +2,22 @@ hqDefine('reports_core/js/bootstrap3/maps', [
     'jquery',
     'underscore',
     'reports/js/bootstrap3/maps_utils',
+    'leaflet',
 ], function (
     $,
     _,
-    mapsUtils
+    mapsUtils,
+    L
 ) {
+    // Reset images needed for map markers, which don't play well with webpack.
+    // See https://github.com/Leaflet/Leaflet/issues/4968#issuecomment-483402699
+    delete L.Icon.Default.prototype._getIconUrl;
+    L.Icon.Default.mergeOptions({
+        iconRetinaUrl: require('leaflet/dist/images/marker-icon-2x.png'),
```

Comment:
What are the `'require' is not defined` lint warnings about? Can eslint be configured to recognize those as globals or is there a better way to handle that?

Reply:
The right way to handle it is to update the ECMAScript version in eslint's config file. I can do that in a followup.

Reply:
https://github.com/dimagi/commcare-hq/pull/35651